core/src/main/java/tech/tablesaw/columns/dates/PackedLocalDate.java

```diff
@@ -362,7 +362,7 @@ public static int plusYears(int yearsToAdd, int packedDate) {
     byte m = getMonthValue(packedDate);
     short y = getYear(packedDate);
 
-    int newYear = YEAR.checkValidIntValue(yearsToAdd + y);
+    int newYear = YEAR.checkValidIntValue((long)yearsToAdd + y);
```

Comment:
```suggestion
    int newYear = YEAR.checkValidIntValue((long) yearsToAdd + y);
```

Reply:
@ccleva if you allow edits from maintainers I could merge this PR

Reply:
Oh I completely missed that option, will do thanks